kernel-python/declarativewidgets/urth_widget.py

```diff
@@ -51,15 +52,19 @@ def send_status(self, status, msg=""):
             }
         })
 
-    def error(self, msg):
+    def error(self, msg, stack_trace=None):
```

Comment:
What do you think about moving all the traceback inside this function. That way we don't have to import and send a stack_trace everywhere `error` is called. Less changes.